### ISSUE TITLE
Recording doesn't work unless i launch TF2 through the program

### ISSUE THREAD

**Issue by Julez7** [AronParker/AviRecorder#10]
If i launch the game with lawena F9 doesn't do anything, even though the console says it's bound to "avi_recorder_startstop"

**Comment by AronParker** [comment]
Did you try to untick the start game checkbox at the bottom?

**Comment by Julez7** [comment]
Yes, when it's ticked and i launch the game with it everything works
If it's unticked and i launch with lawena or manually i can't record

**Comment by AronParker** [comment]
Do the TGA frames get deleted? What do you mean by "I can't record", what exactly isn't happening that stops the recording?

**Comment by Julez7** [comment]
Literally nothing happens when i press F9, it doesn't even start making the TGA frames

**Comment by AronParker** [comment]
Does something print when you press F9? Try typing alias into the console to see what its bound to, try executing it manually, does that work? It should execute a file, does that file exist?

**Comment by Julez7** [comment]
Nothing prints and typing the alias in console doesn't do anything as well. I guess i'm just gonna do all the lawena configs manually
